functions/consumer/rabbit-consumer/src/main/java/org/springframework/cloud/fn/consumer/rabbit/RabbitConsumerConfiguration.java

```diff
@@ -47,16 +48,21 @@
 import org.springframework.messaging.Message;
 import org.springframework.messaging.MessageHandler;
 
+/**
+ * A configuration for RabbitMQ Consumer function. Uses a
+ * {@link AmqpOutboundChannelAdapterSpec} to save payload contents to RabbitMQ.
+ *
+ * @author Soby Chako
+ * @author Nicolas Labrot
+ * @author Chris Bono
+ */
 @EnableConfigurationProperties(RabbitConsumerProperties.class)
 @Configuration
 public class RabbitConsumerConfiguration implements DisposableBean {
 
 	@Autowired
 	private RabbitProperties bootProperties;
 
-	@Autowired
-	private ObjectProvider<ConnectionNameStrategy> connectionNameStrategy;
```

Comment:
This was only in here to appease the signature of the connection factory creator. The code immediately turned arounds and overrides the naming strategy so I removed this.